### PR TITLE
feat!: Add blog back to main repository

### PR DESCRIPTION
**Pull request by jonaharagon**
Changes proposed in this PR:

- Moves the blog back here, so that people feel more comfortable updating old articles
- Renames Blog to Articles

@dngray if you merge this PR please don't push a release afterward, until I'm ready to redirect old blog links to the new URLs :)

<summary>

- [x] I agree to the terms listed below:
  <details><summary>Contribution terms (click to expand)</summary>
  1) I am the sole author of this work.
  2) I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
  3) I have disclosed any relevant conflicts of interest in my post.
  4) I agree to the Community Code of Conduct.
  </details>

### CODE REVIEW

**Review by kimg45 — APPROVED** [review]

**Comment by blacklight447** [comment]
Does this break existing links to the blogs?

**Comment by jonaharagon** [comment]
> Does this break existing links to the blogs?

This was already answered in the OP

**Review by blacklight447 — APPROVED** [review]

File: blog/posts/mozilla-disappoints-us-yet-again-2.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+## Lack of Consent
+
+Mozilla constantly fails to understand the basic concept of consent. Firefox developers seem to see their position as shepherds, herding the uninformed masses towards choices they interpret to be "good for them." Firefox users are not a captive audience that needs to be coddled, they are generally full-grown adult computer users who need to be listened to.
+
+One Mozilla developer claimed that explaining PPA would be too challenging, so they had to opt users in by default.
+
+<iframe src="https://mastodon.social/@Schouten_B/112784434152717689/embed" class="mastodon-embed" style="max-width: 100%; border: 0" width="400" allowfullscreen="allowfullscreen"></iframe><script src="https://mastodon.social/embed.js" async="async"></script>
+
+<iframe src="https://mastodon.social/@Schouten_B/112784608473016028/embed" class="mastodon-embed" style="max-width: 100%; border: 0" width="400" allowfullscreen="allowfullscreen"></iframe><script src="https://mastodon.social/embed.js" async="async"></script>
```
Comment (redoomed1) [inline blog/posts/mozilla-disappoints-us-yet-again-2.md:41]: This feature doesn't display correctly in the preview. I see two white rectangles on this part of the page when looking at it on Mullvad Browser.

File: blog/posts/signal-configuration-and-hardening.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+    - Tutorials
+tags:
+    - Signal
+    - Molly
+    - Instant Messengers
+license: BY-SA
+---
+# Signal Configuration and Hardening Guide
+
+[Signal](https://www.privacyguides.org/real-time-communication#signal) is a widely regarded instant messaging service that is not only easy to use but is also private and secure. Signal's strong E2EE implementation and metadata protections provide a level of assurance that only you and your intended recipients are able to read communications.<!-- more -->
```
Comment (redoomed1) [inline blog/posts/signal-configuration-and-hardening.md:19]:
```suggestion
[Signal](https://www.privacyguides.org/en/real-time-communication#signal) is a widely regarded instant messaging service that is not only easy to use but is also private and secure. Signal's strong E2EE implementation and metadata protections provide a level of assurance that only you and your intended recipients are able to read communications.<!-- more -->
```

File: blog/posts/signal-configuration-and-hardening.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+
+This guide details actions you can take to configure and harden Signal in accordance with your [threat model](https://www.privacyguides.org/basics/threat-modeling/).
```
Comment (redoomed1) [inline blog/posts/signal-configuration-and-hardening.md:21]:
```suggestion
This guide details actions you can take to configure and harden Signal in accordance with your [threat model](https://www.privacyguides.org/en/basics/threat-modeling/).
```

File: blog/posts/signal-configuration-and-hardening.md [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+    The server will not enforce the registration lock after 7 days of inactivity. After that, someone will be able to reset the PIN at registration and register with your phone number. This will wipe the data stored in your Signal account, as it is encrypted by the PIN, but it won't prevent someone from registering with your number provided that they can receive a text on it.
+
+**Important update**: since this blog post was published, there have been changes to the registration flow for Signal. You should read about this [here](signal-number-registration-update.md).
+
+If you haven't set up a Signal PIN, or have previously opted out of setting one up, follow these steps on Android/iOS:
+
+- Select :material-dots-vertical: > **Settings** > **Account** > **Signal PIN**
+- Select **Create new PIN**
+
+Signal will prompt you to enter a PIN. We suggest using a strong alphanumeric PIN that can be stored in a [password manager](https://www.privacyguides.org/passwords/).
```
Comment (redoomed1) [inline blog/posts/signal-configuration-and-hardening.md:42]:
```suggestion
Signal will prompt you to enter a PIN. We suggest using a strong alphanumeric PIN that can be stored in a [password manager](https://www.privacyguides.org/en/passwords/).
```

File: blog/posts/signal-configuration-and-hardening.md [hunk 5]
```diff
@@ -0,0 +1,274 @@
...
+
+If your threat model calls for it, avoid linking your Signal account to a desktop device to reduce your attack surface.
+
+### Endpoint Security
+
+Signal takes security very seriously, however there is only so much an app can do to protect you.
+
+It is very important to take device security on both ends into account to ensure that your conversations are kept private.
+
+We recommend an up-to-date [GrapheneOS](https://www.privacyguides.org/android#grapheneos) or iOS device.
```
Comment (redoomed1) [inline blog/posts/signal-configuration-and-hardening.md:210]:
```suggestion
We recommend an up-to-date [GrapheneOS](https://www.privacyguides.org/en/android/distributions#grapheneos) or iOS device.
```

File: blog/posts/signal-configuration-and-hardening.md [hunk 6]
```diff
@@ -0,0 +1,274 @@
...
+1. If you become aware that your device has been compromised, you should not unlock Molly's database.
+
+If both of the above conditions are met, the data within Molly is safe as long as the passphrase is not accessible to the attacker.
+
+To supplement the database encryption feature, Molly securely wipes your device's RAM once the database is locked to defend against forensic analysis.
+
+While Molly is running, your data is kept in RAM. When any app closes, its data remains in RAM until another app takes the same physical memory pages. That can take seconds or days, depending on many factors. To prevent anyone from dumping the RAM to disk and extracting your data after Molly is locked, the app overrides all free RAM memory with random data when you lock the database.
+
+There is also the ability to configure a SOCKS proxy in Molly to route its traffic through the proxy or [Tor via Orbot](https://www.privacyguides.org/tor#orbot). When enabled, all traffic is routed through the proxy and there are no known IP or DNS leaks. When using this feature, [call relaying](#call-relaying) will always be enabled, regardless of the setting.
```
Comment (redoomed1) [inline blog/posts/signal-configuration-and-hardening.md:262]:
```suggestion
There is also the ability to configure a SOCKS proxy in Molly to route its traffic through the proxy or [Tor via Orbot](https://www.privacyguides.org/en/tor#orbot). When enabled, all traffic is routed through the proxy and there are no known IP or DNS leaks. When using this feature, [call relaying](#call-relaying) will always be enabled, regardless of the setting.
```

**Review by redoomed1 — COMMENTED** [review]
Here are corrections to broken links on the Signal Configuration post, which is one of the few blog posts linked on the site.

File: blog/posts/mozilla-disappoints-us-yet-again-2.md (hunk 1 above)
Comment (jonaharagon) [reply blog/posts/mozilla-disappoints-us-yet-again-2.md:41]: This will be fixed in the next release, it's a CSP issue server-side, not with this PR.

**Review by redoomed1 — APPROVED** [review]

**Pull request merged**